Source/DayView.swift

```diff
@@ -46,21 +54,49 @@ class DayView: UIView {
     let size = CGSize(width: contentWidth, height: 0)
     timelinePager.contentSize = size
 
-    dayHeaderView.anchorAndFillEdge(.Top, xPad: 0, yPad: 20, otherSize: headerHeight)
+    dayHeaderView.anchorAndFillEdge(.Top, xPad: 0, yPad: 0, otherSize: headerHeight)
     timelinePager.alignAndFill(align: .UnderCentered, relativeTo: dayHeaderView, padding: 0)
   }
 }
 
 extension DayView: PagingScrollViewDelegate {
   func viewRequiresUpdate(view: UIView, viewBefore: UIView) {
+    let timeline = (view as! TimelineContainer).timeline
```

Comment:
What I'm actually would like to achieve is type safety. It's 100% clear that `view` is `TimelineContainer`, but how can I enforce type safety there ?